clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -1484,6 +1484,39 @@ void AddUnalignedAccessWarning(ArgStringList &CmdArgs) {
 }
 }
 
+static void handlePAuthABIOption(const ArgList &DriverArgs,
```

Comment:
I think it will be worth a comment above and possibly inline explaining the ABI implications of the options.

For example:
```
Each combination of options here forms a signing schema, and in most cases each signing schema is its own incompatible ABI. The default values of the options represent the default signing schema.
```

Reply:
I've added the comment, thanks! See fcd090caac9ede6b915db991819298bed4a5d44e